styles/facts.css

```
@@ -0,0 +1,66 @@
+body{
+    margin: 15px;
+    background: #ebebeb;
+    font-family: 'Fira Code', 'Source Code Pro', 'Inconsolata', 'Ubuntu Mono', monospace;
+    line-height: normal;
+}
+
+.nav_container{
+    display: flex;
+    margin: 20px;
+    justify-content: space-between;
+}
+.container {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 1000px;
+    flex-wrap: wrap;
+    justify-content: space-around;
+}
```

Nitpick: blank lines

Typically, I've seen one blank line between rulesets, but I've also heard of some conventions having NO blank lines between rulesets.

Whatever you choose will work but there needs to be consistency within a single file (here I see a mix of no blank lines between rule sets and one blank line between rulesets) and also consistency across every `.css` file in the project. 

```suggestion
.nav_container{
    display: flex;
    margin: 20px;
    justify-content: space-between;
}

.container {
    display: flex;
    flex-direction: row;
    width: 100%;
    height: 1000px;
    flex-wrap: wrap;
    justify-content: space-around;
}
```